cvat/apps/engine/views.py

```diff
@@ -1234,13 +1242,12 @@ def preview(self, request, pk):
 class JobViewSet(viewsets.GenericViewSet, mixins.ListModelMixin,
     mixins.RetrieveModelMixin, PartialUpdateModelMixin, UploadMixin, AnnotationMixin
 ):
-    queryset = Job.objects.select_related('segment__task__data').prefetch_related(
-        'segment__task__label_set', 'segment__task__project__label_set',
-        'segment__task__label_set__sublabels__attributespec_set',
-        'segment__task__project__label_set__sublabels__attributespec_set',
-        'segment__task__label_set__attributespec_set',
-        'segment__task__project__label_set__attributespec_set'
-    ).all()
+    queryset = Job.objects.select_related('assignee', 'segment__task__data',
+        'segment__task__project').annotate(Count('issues', distinct=True),
+        task_labels_count=Count('segment__task__label',
+            filter=Q(segment__task__label__parent__isnull=True), distinct=True),
+        proj_labels_count=Count('segment__task__project__label',
+            filter=Q(segment__task__project__label__parent__isnull=True), distinct=True)).all()
```

Comment:
Could you reformat this like in `ProjectViewSet`? It's hard to understand which arguments are for what.